src/project/types/confluence/confluence.ts

```diff
@@ -0,0 +1,83 @@
+import { ProjectScaffoldFile } from "../types.ts";
+
+export const buildConfluenceFiles = (): ProjectScaffoldFile[] => {
+  const file: ProjectScaffoldFile[] = [
+    {
+      name: "project-roadmap",
+      content: `# Welcome
+Welcome to this sample Quarto Confluence Project! Feel free to use this as a way to help get you started in building your own Confluence project.
```

Comment:
@jjallaire Would you like to see some changes here?

Reply:
Maybe something more like:

"This project provides provides a simple scaffold for creating a Quarto Confluence Project. You'll almost certainly want to remove the sample files, they are just here as examples. When you add you own documents (including ones in subfolders) they will be automatically added to the site navigation sidebar."

Reply:
Also, I'm thinking that we'd want this content in the `index.qmd` ? (let's discuss realtime in our 1:1 today)